docs/caveats.md

```diff
@@ -445,27 +445,52 @@ See [](generic-linux-devices)
 
 (caveats-sros)=
 ## Nokia SR OS
+
 * Only supported on top of *Containerlab*, using VRNetlab (VM running inside container)
-* Requires the latest `nokia.grpc` Ansible Galaxy collection and its dependencies to be installed from the git repo. You can also use the **netlab install grpc** command to install them
+* With the launch of the Nokia SR SIM, we stopped running integration tests for the SR-OS VM. The last release with which we ran a full suite of integration tests for the SR-OS VM was 25.07.
 
-```
-ansible-galaxy collection install git+https://github.com/nokia/ansible-networking-collections.git#/grpc/
-python3 -m pip install grpcio protobuf==3.20.1
-```
+Other caveats (shared with Nokia SR-SIM):
+
+* We use the `nokia.grpc` Ansible Galaxy collection to configure SR-OS. The latest version of that collection has to be installed from the git repo:
+
+  ```
+  ansible-galaxy collection install git+https://github.com/nokia/ansible-networking-collections.git#/grpc/
+  python3 -m pip install grpcio protobuf==3.20.1
+  ```
```

Comment:
was it ever considered to switch from gnmi to netconf for SR OS? I thought it was once discussed?

Reply:
See e.g. https://github.com/napalm-automation-community/napalm-sros

It would be possible, but a significant effort for no functional gain. I initially made the choice to use gnmi, both to have consistency between SR OS and SR Linux and also because I knew that Netconf was the preferred, presumably better tested mechanism, so I figured focusing on the other one could have benefits in terms of potentially finding issues that might otherwise not be found.

People are welcome to contribute